src/pages/Products.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from "react";
+import ProductInfo from "../components/domain/Product/ProductInfo/ProductInfo";
+
+type ProductType = {
+  id: string;
+  price: number;
+  name: string;
+  imageUrl: string;
+};
+
+type ProductsType = ProductType[];
+
+const Products = () => {
+  const [products, setProducts] = useState([
+    { id: "", price: 0, name: "", imageUrl: "" },
+  ]);
+
+  useEffect(() => {
+    fetch("http://localhost:3000/products").then((res) =>
+      res.json().then((res) => {
+        setProducts(res);
+      })
+    );
+  }, []);
+
+  return (
+    <section className="product-container">
+      {products.map((product: ProductType) => (
+        <ProductInfo
```

Comment:
`products` 타입이 잘 선언되었다면 product 타입은 자동적으로 추론되니 타입 선언은 하지 않으셔도 됩니다😄

```suggestion
      {products.map(({ id, name, price }) => (
        <ProductInfo
```

Reply:
[bcb16b3](https://github.com/jlee0505/react-shopping-cart/pull/2/commits/bcb16b3c7aa0fa46a5940ba85764e5392c851184)